### PR TITLE
feat(TDP-2800): breadcrump with dropdown

### PR DESCRIPTION
**Pull request by amaalej**
**Link to the JIRA issue**
e.g. https://jira.talendforge.org/browse/TDP-2800

**Please check if the PR fulfills these requirements**
- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)
There is an opened issue for IE11: https://github.com/Talend/react-talend-components/issues/137

**(Optional) What is the current behavior?**
The ellipsis is not clickable.

**(Optional) What is the new behavior?**
The hidden items (currently the folders) appear now on the dropdown inside the breadcrumb.

### CODE REVIEW

File: dataprep-webapp/src/app/components/widgets-containers/breadcrumb/breadcrumb-container.spec.js [hunk 1]
```diff
@@ -89,5 +137,22 @@ describe('Breadcrumb container', () => {
...
 			expect(SettingsActionsService.dispatch.calls.argsFor(0)[0].payload.id).toBe('abcd');
 		}));
+
+		it('should dispatch button click', inject((SettingsActionsService) => {
+			// given
+			createElement(breadcrumb);
+
+			// when
+			const parentFolderButton = element.find('.breadcrumb .dropdown-menu > li > a').eq(0);
+			parentFolderButton.click((e) => {
```
Comment (jsomsanith) [inline dataprep-webapp/src/app/components/widgets-containers/breadcrumb/breadcrumb-container.spec.js]: What is this code for ?

**Comment by amaalej** [comment]
@jsomsanith for your questions:
*Have you tested on every browser ? (checkbox not checked)*
Yes I checked on every brower, that's how I detected the issue on IE. So no need for the reviewer to redo the test on all browsers.

*Have you worked on the IE bug ? Because it is part of this feature.*
I opened an issue on components.

**Comment by jsomsanith** [comment]
`Yes I checked on every brower, that's how I detected the issue on IE. So no need for the reviewer to redo the test on all browsers.`
--> please update the PR description

`I opened an issue on components.`
--> so ko on TDP, you need to fix the issue, report it is not enough

**Comment by amaalej** [comment]
@jsomsanith done.

**Comment by jsomsanith** [comment]
This feature rely on new common lib version, can you make a PR on EE ? (or this will break if we merge this one alone)

**Comment by amaalej** [comment]
@jsomsanith done: https://github.com/Talend/data-prep-ee/pull/87

**Pull request merged**